### ISSUE TITLE
Hard to read about too long part

### ISSUE THREAD

**Issue by LifHoshi** [LifHoshi/pe#14]
![image.png](https://raw.githubusercontent.com/LifHoshi/pe/main/files/3564ca4f-393b-41a2-9d47-9345a63220f3.png)

![image.png](https://raw.githubusercontent.com/LifHoshi/pe/main/files/802dd66c-b4e6-4b64-b26b-91c1e82ee665.png)

**Comment by nus-pe-script** [comment]
# Team's Response
Thanks for your report. However, we need to mark this issue as Rejected because it doesn't affect any usage of the product and the explanation is necessary for instructing the users about how to use the command and constraints about the commands. There is no unnecessary information in those explanations. We could not neglect important information just to make the UG look shorter and nicer.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** It looks messy and lacks structure. 

They could do like:

For the RIght Part:
                 -  What can this do
                 - What can else do
                 - ....

For the Left Part:
                 - What can this do
                 - what can else do
                 - ...

Instead, put all functions into one giant paragraph. Same issue for the phone, email and address description. 

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Low`]

- [x] I disagree

**Reason for disagreement:** This shouldn't be considered as verylow. 

It is not a tiny cosmetic problem. It is somehow a flaw of UG that too wordy and lengthy. 

<catcher-end-of-segment><hr>
